extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/temporal/AccumuloTemporalIndexer.java

```diff
@@ -158,16 +167,20 @@ public Configuration getConf() {
     private void storeStatement(final Statement statement) throws IOException, IllegalArgumentException {
         // if the predicate list is empty, accept all predicates.
         // Otherwise, make sure the predicate is on the "valid" list
-        final boolean isValidPredicate = validPredicates.isEmpty() || validPredicates.contains(statement.getPredicate());
+        final boolean isValidPredicate = validPredicates == null || validPredicates.isEmpty() || validPredicates.contains(statement.getPredicate());
         if (!isValidPredicate || !(statement.getObject() instanceof Literal)) {
             return;
         }
+
         final DateTime[] indexDateTimes = new DateTime[2]; // 0 begin, 1 end of interval
         extractDateTime(statement, indexDateTimes);
         if (indexDateTimes[0]==null) {
             return;
         }
 
+        if (!this.isInit)
```

Comment:
Indexes should not create their own batchwriter because it does not get flushed properly.
If we call init() here, then the batchwriter may not have been set.  If it did not initialize at this point, then this is a bug and should fail.